multiapps-controller-web/src/main/java/org/cloudfoundry/multiapps/controller/web/configuration/service/ObjectStoreServiceInfoCreator.java

```diff
@@ -80,4 +89,19 @@ private ObjectStoreServiceInfo createServiceInfoForAliCloud(Map<String, Object>
                                               .build();
     }
 
+    private ObjectStoreServiceInfo createServiceInfoForGcpCloud(Map<String, Object> credentials) {
+        String bucket = (String) credentials.get("bucket");
+        String region = (String) credentials.get("region");
+        final byte[] decodedKey = Base64.getDecoder()
+                                        .decode((String) credentials.get("base64EncodedPrivateKeyData"));
+        final String decodedCredential = new String(decodedKey, Charset.forName("UTF-8"));
```

Comment:
is this really needed to be final, or can it be just an effective final?

Reply:
done